### ISSUE TITLE
Uncaught Error: Encountered an invalid scope end id. Popped -1, expected to pop -329.

### ISSUE THREAD

**Issue by phillt** [adambutler/atom-prettify#11]
[Enter steps to reproduce below:]

1. Use media queries 

        @media (max-width: 500px){
             ....
         }

2. Call the prettify command using the `cmd + shift + p` and type *prettify* hit enter and boom! error.

**Atom Version**: 0.205.0
**System**: Mac OS X 10.10.3
**Thrown From**: [atom-prettify](https://github.com/adambutler/atom-prettify) package, v0.1.1

### Stack Trace

Uncaught Error: Encountered an invalid scope end id. Popped -1, expected to pop -329.

```
At /Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:755

Error: Encountered an invalid scope end id. Popped -1, expected to pop -329.
  at TokenizedBuffer.module.exports.TokenizedBuffer.scopesFromTags (/Applications/Atom.app/Contents/Resources/app.asar/src/tokenized-buffer.js:573:21)
  at TokenizedBuffer.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/tokenized-buffer.js:461:31)
  at TokenizedBuffer.module.exports.TokenizedBuffer.buildTokenizedLinesForRows (/Applications/Atom.app/Contents/Resources/app.asar/src/tokenized-buffer.js:468:10)
  at TokenizedBuffer.module.exports.TokenizedBuffer.handleBufferChange (/Applications/Atom.app/Contents/Resources/app.asar/src/tokenized-buffer.js:369:32)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/tokenized-buffer.js:62:24
  at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
  at TextBuffer.module.exports.TextBuffer.applyChange (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:624:20)
  at TextBuffer.module.exports.TextBuffer.setTextInRange (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:532:12)
  at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:516:26
  at TextBuffer.module.exports.TextBuffer.transact (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:750:18)
  at TextBuffer.module.exports.TextBuffer.setTextInRange (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:514:21)
  at TextEditor.module.exports.TextEditor.setTextInBufferRange (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:710:31)
  at /Users/philllware/.atom/packages/atom-prettify/lib/prettify.coffee:16:12
  at Array.forEach (native)
  at prettify (/Users/philllware/.atom/packages/atom-prettify/lib/prettify.coffee:12:18)
  at atom-text-editor.atom.commands.add.prettify:prettify (/Users/philllware/.atom/packages/atom-prettify/lib/prettify.coffee:8:7)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:238:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at CommandPaletteView.module.exports.CommandPaletteView.confirmed (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/command-palette/lib/command-palette-view.js:159:32)
  at CommandPaletteView.module.exports.SelectListView.confirmSelection (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:338:21)
  at space-pen-div.atom.commands.add.core:confirm (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:109:19)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:238:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:520:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:355:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:177:20)

```

### Commands

```
     -5:24.8.0 go-to-line:toggle (atom-text-editor.editor.is-focused)
     -5:22.8.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -5:17.6.0 core:paste (atom-text-editor.editor.is-focused)
     -5:15.5.0 core:save (atom-text-editor.editor.is-focused)
  2x -0:04.2.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:03.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:01.4.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:01.3.0 prettify:prettify (atom-text-editor.editor)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "jsformat",
      "autocomplete-snippets"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-beautify, v0.27.10
atom-prettify, v0.1.1
color-picker, v2.0.4
css-comb, v2.0.4
highlight-selected, v0.9.3
language-blade, v0.14.0
language-generic-config, v0.2.0
laravel, v0.7.0
laravel-facades, v1.0.0
linter, v0.12.6
ssh-config, v0.10.1

# Dev
No dev packages
```

**Comment by adambutler** [comment]
Seems to be unintentionally fixed by #14. @phillt can you confirm this now works in `0.1.3` ?

**Comment by phillt** [comment]
Works great @adambutler! Thanks!
